spec/Records.tex

```diff
@@ -186,23 +185,18 @@ \subsection{Record Initialization}
 is initialized through a call to the record's default initializer,
 passing no arguments.  The default initializer for a record is defined in
```

Comment:
This doesn't seem right to me. The compiler will call whatever record initializer accepts 0 arguments, whether or not it's one the compiler generated.

Maybe

> A variable of a record type declared without an initialization expression is initialized with a call to the initializer for that record that accepts zero arguments. If multiple such initializers exist, an ambiguity error is raised.